examples/multi_inspiral/run.sh

```diff
@@ -65,7 +66,7 @@ pycbc_multi_inspiral \
     --psd-segment-length 32 \
     --psd-segment-stride 8 \
     --psd-num-segments 29 \
-    --num-slides 1 \
+    --do-shortslides \
```

Comment:
Ho much longer is this test?  If you see that it lasts "too long" increase the `slide-shift` as needed, to keep the runtime under control.   It's okay to do for testing purposes.

Reply:
Hi @pannarale I run the multi_inspiral example on a Macbook air with M1 processor and here are the results

No slides 39s
`2024-05-30T12:25:19.805+02:00 INFO : Clustering slide 0
2024-05-30T12:25:19.805+02:00 INFO : Filtering completed
2024-05-30T12:25:19.805+02:00 INFO : Writing output
2024-05-30T12:25:19.821+02:00 INFO : Finished
2024-05-30T12:25:19.821+02:00 INFO : Time to complete analysis: 39

`

Turning on slides with 3ifos ,--segment-length 256 and --slide-shift=1, the equation without the numIfo factor gives 65 slides. It takes 45 seconds
`2024-05-30T12:26:27.767+02:00 INFO : Clustering slide 64
2024-05-30T12:26:27.768+02:00 INFO : Clustering slide 65
2024-05-30T12:26:27.772+02:00 INFO : Filtering completed
2024-05-30T12:26:27.772+02:00 INFO : Writing output
2024-05-30T12:26:27.843+02:00 INFO : Finished
2024-05-30T12:26:27.843+02:00 INFO : Time to complete analysis: 45`

Reply:
Great, thanks!